part-spec/pinPaths.json

```diff
@@ -0,0 +1,42 @@
+{
+  "$id": "https://github.com/edatasheets/edatasheets.github.io/blob/main/part-spec/pinPaths.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "pinPaths": {
+    "type": "object",
+    "required": [
+      "partPinPaths"
+    ],
+    "properties": {
+      "numberofpinPaths": {
+        "description": "number of pinPaths defined. This number should not be higher than the number of components included in the part",
+        "type": "number"
+      },
+      "partPinPaths": {
+        "description": "list of pins associated with each component in a part. This is when a part includes multiple components",
```

Comment:
reword suggestion:

list of pins associated with each component in a multi-component part

Reply:
changed